### DISCUSSION TITLE
About Admin home page

### DISCUSSION THREAD

**Discussion by su-network** [aminalaee/sqladmin#352] [General]
I think there should be a list of **table name** at _home page_ so when someone **redirects** to http://localhost:8000/admin/ it helps them to look table easily.

**Comment by aminalaee** [comment]
Hey, thanks for the suggestion, I think it makes sense, can you please explain that by home page you mean / ? or /admin ?

**Reply by su-network** [reply]
I am talking about /admin,that needs to design in such a way that if user land at that page it can find all the **table names** .

**Reply by aminalaee** [reply]
but each table name that is installed with the Admin, is shown in the side bar, how is it any different?

**Reply by su-network** [reply]
Yes that's correct but we can make use of dashboard other than table names we can add more content instead of empty page.

**Reply by aminalaee** [reply]
I've assumed you can override it to have custom views, but maybe it's useful to provide some basic info there.

**Reply by su-network** [reply]
What you think about having user ability to create SQL alchemy models on database  and manage it on admin page.🤔

**Comment by iagobalmeida** [comment]
Its pretty simple to override the template, just create a `templates/index.html` file in your project, then start by placing the Jinja extends and block tags:
```html
{% extends "layout.html" %}
{% block content %}
<!-- Place HTML here -->
{% endblock %}
```

I've create a welcome page with some helpfull links to other features that are implemented in the admin, using the Tabler.css documentation it's pretty easy to create beatifull pages:
![image](https://user-images.githubusercontent.com/35234046/230435961-2d3c6602-421b-41d2-957e-c6ca8d66e665.png)
```html
{% extends "layout.html" %}
{% block content %}
    <div class="col-12 col-xxl-6">
        <div class="card">
            <div class="card-header">
                <h3 class="card-title">
                    <i class="fa fa-star me-2"></i>
                    Bem vindo!
                </h3>
            </div>
            <div class="card-body border-bottom py-3">
                <p>É possível acessar as tabelas do Dataside através de APIs, para gerenciar as integrações, acesse a página de integração <a href="http://localhost:8000/admin/integracao">clicando aqui</a>.</p>
            </div>
        </div>
    </div>
{% endblock %}
```

The Tabller.css is already "included" in the project, so you don't need to worry about importing it or anything. Heres the complete documentation: https://tabler.io/docs/getting-started
